xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/macros.vm

```diff
@@ -952,15 +952,15 @@ $html
 ###
 
 #macro(warning $text)
-<div class="warningmessage"><span class="messagetype">$services.localization.render('warning'): </span>$text</div>
+<div class="box warningmessage"><span class="messagetype">$services.localization.render('warning'): </span>$text</div>
 #end
 
 #macro(error $text)
-<div class="errormessage"><span class="messagetype">$services.localization.render('error'): </span>$text</div>
+<div class="box errormessage"><span class="messagetype">$services.localization.render('error'): </span>$text</div>
 #end
 
 #macro(info $text)
-<div class="infomessage">$text</div>
+<div class="box infomessage">$text</div>
```

Comment:
We applied changes to the style of the boxes. When generated using the macro, all the message types have the box type by default so I assumed I could factorize some of the style to get onto the `.box` class. It makes sense to do so IMO. See xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/messages.less

These velocity macro did not generate the boxes exactly the same as the XWiki macro and the lack of the `box` class would break the style. With this change the few use of those velocity macros should match the normal style :)

Another solution would be to rework the selectors in `messages.less` to take into account the cases where there is no `box` class. I'd rather have simpler LESS and more consistency.

Let me know if you think we should move onto the other solution :)

Reply:
Sounds good.